dj_cqrs/dataclasses.py

```diff
@@ -73,18 +113,47 @@ def previous_data(self):
     def correlation_id(self):
         return self.__correlation_id
 
+    @property
+    def expires(self):
+        return self.__expires
+
+    @property
+    def retries(self):
+        return self.__retries
+
+    @retries.setter
```

Comment:
Previously this class was immutable - is it ok to do it mutable now @maxipavlovic ?

Reply:
I see no real problems with that :)